### PR TITLE
open-cluster-management/backlog#3378 -- kui review

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: console/vwt_search.adoc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [#visual-web-terminal]
 = Visual Web Terminal
 
-The Visual Web Terminal is a *tech preview* feature. You can use the Visual Web Terminal to run many commands across your environment. Run commands and get outputs without leaving the entire console. You can run multiple commands.
+The Visual Web Terminal is a *tech preview* feature. You can use the Visual Web Terminal to run many commands across your cluster. Run commands and get outputs without leaving the entire console. You can run multiple commands, similar to what you can do in a standard terminal.
 
-For instance, when categorized data is returned, such as when you enter a `search` command, it is returned in an interactive tabular format.
+With Visual Web Terminal, many commands that return cluster resource data are rendered visually in interactive tabular format rather than remndered in just plain text. You can see the data, click on a row, and then see more details about that resource.
```
Comment (chrisahl) [inline console/vwt_search.adoc]: Leaving out the `search`command example as well as not adding the more familiar `kubectl` and `oc` commands I suggested earlier today is fine with me.   This keeps it more generic.

But I would correct the typo for `remndered`

Comment (swopebe) [reply console/vwt_search.adoc]: sorry, that made it in another commit, I just had to meet with mark and push it. Please refresh to see latest?

**Review by chrisahl — APPROVED** [review]

File: console/vwt_search.adoc [hunk 2]
```diff
@@ -24,9 +24,10 @@ You can access the Visual Web Terminal by completing the following steps:
 
 . Log in to your console.
 . From the console _Header_, select the *Visual Web Terminal* icon.
-. Choose to open in a new tab or open in a current tab.
-. Select the `getting started` command.
-. Optional: Run `oc --namespace=<your-namespace>` to set a default namespace to run commands only within that namespace. 
+. Choose to open in a new tab or open in a current tab. 
+. Start running commands in the status bar. For example, you can start running any `oc` or `kubectl` commands and get a visual output.
```
Comment (mark-nc) [inline console/vwt_search.adoc]: The user does not run commands in the status bar.  The status bar is under the command input field.  The user runs commands in the command input field.  (We were talking about the status bar below when talking about the button/widget you can click to change the default namespace.)

File: console/vwt_search.adoc [hunk 3]
```diff
@@ -24,9 +24,10 @@ You can access the Visual Web Terminal by completing the following steps:
...
+. Optional: Select the `getting started` command to pull up the _About_ page and more, or type the command in the command status.
+. Optional: You can change your default namespace. Click `@default` to run the command that displays your namespaces. Click on a row in the table to set a different namespace.
```
Comment (mark-nc) [inline console/vwt_search.adoc]: This is where reference to the status bar would come in.

**Review by mark-nc — COMMENTED** [review]
See inline comments below.

**Review by mark-nc — APPROVED** [review]
lgtm

**Comment by mark-nc** [comment]
I am OK with closing this now.

**Pull request merged**